src/com/gh4a/adapter/IssueEventAdapter.java

```diff
@@ -170,6 +179,59 @@ protected Intent getIntent() {
         }
     }
 
+    @Nullable
+    private Drawable getIssueEventIcon(final IssueEvent event) {
+        int iconAttr;
+
+        switch (event.getEvent()) {
+            case IssueEvent.TYPE_CLOSED:
+                iconAttr = R.attr.issueEventClosedIcon;
+                break;
+            case IssueEvent.TYPE_REOPENED:
+                iconAttr = R.attr.issueEventReopenedIcon;
+                break;
+            case IssueEvent.TYPE_MERGED:
+                iconAttr = R.attr.issueEventMergedIcon;
+                break;
+            case IssueEvent.TYPE_REFERENCED:
+                iconAttr = R.attr.issueEventReferencedIcon;
+                break;
+            case IssueEvent.TYPE_ASSIGNED:
+                iconAttr = R.attr.issueEventAssignedIcon;
+                break;
+            case IssueEvent.TYPE_UNASSIGNED:
+                iconAttr = R.attr.issueEventUnassignedIcon;
+                break;
+            case IssueEvent.TYPE_LABELED:
+                iconAttr = R.attr.issueEventLabeledIcon;
+                break;
+            case IssueEvent.TYPE_UNLABELED:
+                iconAttr = R.attr.issueEventUnlabeledIcon;
+                break;
+            case IssueEvent.TYPE_LOCKED:
+                iconAttr = R.attr.issueEventLockedIcon;
+                break;
+            case IssueEvent.TYPE_UNLOCKED:
+                iconAttr = R.attr.issueEventUnlockedIcon;
+                break;
+            case IssueEvent.TYPE_MILESTONED:
+                iconAttr = R.attr.issueEventMilestonedIcon;
+                break;
+            case IssueEvent.TYPE_DEMILESTONED:
+                iconAttr = R.attr.issueEventDemilestonedIcon;
+                break;
+            case IssueEvent.TYPE_RENAMED:
+                iconAttr = R.attr.issueEventRenamedIcon;
+                break;
+
+            default:
+                return null;
+        }
+
+        int iconResId = UiUtils.resolveDrawable(mContext, iconAttr);
+        return ContextCompat.getDrawable(mContext, iconResId);
+    }
```

Comment:
I wonder whether it would make sense to move the lookup into a static final HashMap<String, Integer>, so the binding code would become:

```
int eventIconAttr = EVENT_ICONS.get(event.getEventType());
if (eventIconAttr != 0) {
    holder.ivIssueEvent.setImageResource(UiUtils.resolveDrawable(mContext, eventIconAttr));
    holder.ivIssueEvent.setVisibility(View.VISIBLE);
} else {
    holder.ivIssueEvent.setVisibility(View.GONE);
}
```